src/RelayClient.ts

```diff
@@ -424,6 +418,158 @@ class RelayClient extends EnvelopingEventEmitter {
       estimatedGasCorrectionFactor
     );
   }
+
+  public async relayTransaction(
+    envelopingRequest: UserDefinedEnvelopingRequest,
+    requestConfig: RequestConfig
+  ): Promise<Transaction> {
```

Comment:
Would it be possible to return TransactionResponse instead? https://docs.ethers.org/v5/api/providers/types/#providers-TransactionResponse

Reply:
its possible, based in our conversation but there is a chance that could return undefined so the typing will need to include it.

Reply:
Actually, disregard this. I can wait for the trx receipt by using the provider anyway.